django_mongodb/utils.py

```diff
@@ -18,3 +24,89 @@ def check_django_compatability():
             f"You must use the latest version of django-mongodb {A}.{B}.x "
             f"with Django {A}.{B}.y (found django-mongodb {__version__})."
         )
+
+
+class CollectionDebugWrapper:
```

Comment:
Additionally, this part may not be necessary because [we will have standardized logging](https://pymongo.readthedocs.io/en/stable/examples/logging.html) in our MongoClient. It captures the call operation details.

I'll say leave it for now as

Reply:
As you saw, the first commit is https://github.com/mongodb-labs/django-mongodb/pull/25. It's a dependency of this PR  but I'll merge the other PR first so it'll no longer appear here.